runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -500,13 +507,17 @@ private LootTrackerBox buildBox(LootTrackerRecord record)
 
 		// Create box
 		final LootTrackerBox box = new LootTrackerBox(itemManager, record.getTitle(), record.getType(), record.getSubTitle(),
-			hideIgnoredItems, config.priceType(), config.showPriceType(), plugin::toggleItem);
+			hideIgnoredItems, config.priceType(), config.showPriceType(), plugin::toggleItem, plugin::toggleEvent, plugin.isEventIgnored(record.getTitle()));
```

Comment:
Just call `isEventIgnored` once and store the result

Reply:
I don't understand what you are asking for me to change exactly.

I did notice `LootTrackerBox` could just use the `ignoredEvent` flag locally instead of storing the result, did you mean this?

Reply:
`isEventIgnored` is called on L485 and again here on L510. You can just call it once and store the result in a variable.